draft-ietf-masque-connect-udp.md

```diff
@@ -179,8 +179,15 @@ UDP proxying responses also do not carry any message content.
 
 ## UDP Proxy Handling {#handling}
 
-Upon receiving a UDP proxying request, the recipient UDP proxy extracts the
-"target_host" and "target_port" variables from the URI it has reconstructed
+Upon receiving a UDP proxying request, the recipient can either act as a UDP
+proxy and establish a tunnel (see below), or act as an intermediary and forward
+the request to another HTTP server. Note that such intermediaries may need to
+reencode the request if they forward it using a version of HTTP that is
+different from the one used to receive it, as the request encoding differs by
+version (see below).
```

Comment:
I'd be inclined to more liberally borrow from https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#rfc.section.9.3.6.p.6 and say something like 

UDP proxying requests are intended to be used by a proxy. The recipient can establish a UDP tunnel either by directly establishing a UDP socket to the the request target or, if configured to use another proxy, by forwarding the request to the next inbound proxy. Note that forwarding requests may require them to be reencoded in order to meet the requirements of the HTTP version (see below).

Reply:
That's the text that I started from, but I did some amount of surgery to avoid having to change the rest of the section, and that's how I landed with the current PR. I feel that this text doesn't flow well with the rest of the section. In particular, the term "inbound proxy" isn't defined in this document and we have readers who are not overly familiar with the core HTTP documents so I tried to create something that limits its use of terminology and sets the stage for the rest of the section. What do you think?

Reply:
that's fair. If you force me to think, my minor complaint is the phrase "act as an intermediary and forward" that seems less precise than " if configured to use another proxy".

I've implemented this behaviour, it's weird. It's not something I expect most proxy implementers to think about. Other kinds of deployment (like a cloud environment) are a bit more opaque in their treatment of this kind of forwarding and will likely have a bunch of harder questions to answer if they want to do; so leave them to it.

Reply:
That makes sense. I tweaked the text to incorporate "if configured to use another proxy". What do you think about the latest?